ops/terraform/services/pipeline/modules/bfd_pipeline_dashboard/templates/pipeline_dashboard.json.tftpl

```diff
@@ -0,0 +1,207 @@
+                "background": "transparent"
+            }
+        },
+        {
+            "height": 8,
+            "width": 12,
+            "y": 1,
+            "x": 12,
+            "type": "metric",
+            "properties": {
+                "metrics": [
+                    [ { "expression": "FILL(m1, REPEAT)", "label": "Expression7", "id": "e7", "region": "us-east-1", "visible": false } ],
+                    [ { "expression": "FILL(m2, REPEAT)", "label": "Expression9", "id": "e9", "region": "us-east-1", "visible": false } ],
+                    [ { "expression": "IF(e9 > e7, 1, 0)", "label": "Ongoing Data Load?", "id": "e5", "region": "us-east-1" } ],
+                    [ "bfd-${env}/bfd-pipeline", "time/data-fully-loaded", { "id": "m1", "region": "us-east-1", "visible": false } ],
+                    [ ".", "time/data-first-available", { "id": "m2", "region": "us-east-1", "visible": false } ]
+                ],
+                "view": "timeSeries",
+                "region": "us-east-1",
+                "stat": "Maximum",
+                "period": 60,
+                "title": "Weekly Data Load Status",
+                "yAxis": {
+                    "left": {
+                        "min": -1,
+                        "max": 2,
+                        "showUnits": false
+                    },
+                    "right": {
+                        "showUnits": false
+                    }
+                },
+                "annotations": {
+                    "horizontal": [
+                        {
+                            "color": "#dfb52c",
+                            "label": "Data Load is Running",
+                            "value": 1
+                        },
+                        {
+                            "color": "#00819c",
+                            "label": "Data Load is Finished",
+                            "value": 0
+                        }
+                    ],
+                    "vertical": [
+                        {
+                            "label": "Alert Threshold (9 AM EST/EDT)",
+                            "value": "${next_monday_9am_est}"
+                        },
+                        {
+                            "label": "Alert Threshold (9 AM EST/EDT)",
+                            "value": "${next_next_monday_9am_est}"
+                        },
+                        {
+                            "label": "Alert Threshold (9 AM EST/EDT)",
+                            "value": "${current_week_monday_9am_est}"
+                        },
+                        {
+                            "label": "Alert Threshold (9 AM EST/EDT)",
+                            "value": "${prev_monday_9am_est}"
+                        },
+                        {
+                            "label": "Alert Threshold (9 AM EST/EDT)",
+                            "value": "${prev_prev_monday_9am_est}"
+                        }
```

Comment:
These vertical annotations are great! Given some of the limitations you've run into, these five selected thresholds seem reasonable. 👍